op-node/cmd/stateviz/main.go

```diff
@@ -42,7 +42,7 @@ type SnapshotState struct {
 	L1Head          eth.L1BlockRef `json:"l1Head"`          // what we see as head on L1
 	L1Current       eth.L1BlockRef `json:"l1Current"`       // l1 block that the derivation is currently using
 	L2Head          eth.L2BlockRef `json:"l2Head"`          // l2 block that was last optimistically accepted (unsafe head)
-	L2SafeHead      eth.L2BlockRef `json:"l2SafeHead"`      // l2 block that was last derived
+	L2SafeHead      eth.L2BlockRef `json:"l2Safe"`          // l2 block that was last derived
```

Comment:
should we change the name of the variable too?

Reply:
good find. Yeah, the variable name was changed to `l2Safe`.

Reply:
cool, updated in #d5bad28